### ISSUE TITLE
Document use of Lore for Slides

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#7375]
|<img alt="Nafai's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">| Nafai reported|
|-|-|
|Trac ID|trac#177|
|Type|enhancement|
|Created|2003-08-06 10:54:01Z|

<details><summary>Searchable metadata</summary>

```
trac-id__177 177
type__enhancement enhancement
reporter__Nafai Nafai
priority__normal normal
milestone__ 
branch__ 
branch_author__ 
status__closed closed
resolution__fixed fixed
component__ 
keywords__ 
time__1060167241000000 1060167241000000
changetime__1468803338025111 1468803338025111
version__ 
owner__Nafai Nafai
cc__Nafai cc__spiv
```
</details>

**Comment by twisted-trac** [comment]
|<img alt="Nafai's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">|Nafai commented|
|-|-|

```
#!html
<pre>
Never remember how to produce slides with lore, especially
if you want output in prosper or other formats.

</pre>
```

**Comment by twisted-trac** [comment]
|<img alt="Nafai's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">|Nafai commented|
|-|-|

```
#!html
<pre>
Also, it would be nice if possible input and output formats
would be listed in lore --help

</pre>
```

**Comment by twisted-trac** [comment]
|<img alt="Nafai's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">|Nafai commented|
|-|-|

```
#!html
<pre>
Since it looks like spiv is busy, and I'm been given the
documentation hat this week, I'll do this too.

</pre>
```

**Comment by twisted-trac** [comment]
|<img alt="Nafai's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">|Nafai commented|
|-|-|

```
#!html
<pre>
Let spiv check this over and then close.

</pre>
```

**Comment by twisted-trac** [comment]
|<img alt="spiv's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">|spiv commented|
|-|-|

```
#!html
<pre>
Looks good.  Thanks!

</pre>
```

**Comment by twisted-trac** [comment]
|<img alt="GitHub's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50"><a name="note_6"></a>|GitHub commented|
|-|-|

In changeset 1230f84611c055d76e1b19f0afa096809ca3bbea
```
#!CommitTicketReference repository="" revision="1230f84611c055d76e1b19f0afa096809ca3bbea"
Merge pull request #177 from rodrigc/8457-rodrigc-types

Author: rodrigc
Reviewer: markrwilliams
Fixes: #8457

Replace use of types constants with primitive types in twisted/spread
```
